### ISSUE TITLE
Handle more gerrit events

### ISSUE THREAD

**Issue by vthib** [Intersec/qbot#8]
Some use-cases that we want to develop:

- Notify people when they are set as reviewer of a commit (on first push, and explicitly). Good for small projects, not so much for bigger ones: must be activable per project.
- Notify people that reviewed a commit, when the owner replies to their reviews. Not easy to implement without useless notifications, one solution would be to replicate the behavior of emails (iirc):
  - Do not notify reviewers of a patch, unless they did at least one review
  - Notify those reviewers on every update of the commit (regardless of who made the update)
  Basically, when reviewing a commit, the reviewer indicates an interest on the commit, and receives all subsequents updates.
